index.html

```diff
@@ -240,6 +248,62 @@ <h2 class="my-project-name">Multi-Post Stories</h2>
       </div>
     </div>
   </section>
+
+  <div class="contact-form">
+
+      <h3 class="contact-form-title">CONTACT US</h3>
+
+      <p class="contact-form-title">If you have an application you are interested in developing, a feature that you
+         need built or a project that needs coding. I’d love to help with it</p>
+
+    <form action="https://formspree.io/f/moqrjakr" method="post">
+      
+         <input type="text" name="user-first-name" placeholder="Text Input" maxlength="30" required>
+      
+         <input type="email" name="user-email" placeholder="jeremy@gmail.com" class="name-ph-change" 
+         pattern="/^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/" required>
+       
+         <textarea name="user-message" placeholder="Enter text here" class="name-ph-text-area-change" maxlength="500" required></textarea>
+
+        <button type="submit">Send</button>
+
+
+     </form>
+
+  </div>
+
+  <!-- <img
+    src="./images/Rectangle_42.svg"
+    class="footer-rectangle-separator"
+    alt="Section separator image"
+  /> -->
```

Comment:
- [ ] As I mentioned earlier, please make sure to remove unnecessary code from your codebase.

Reply:
Done! thanks for the hint.